app/views/admin/deploy_groups/show.html.erb

```diff
@@ -40,7 +40,17 @@
                     class: "btn btn-default"
         %>
         <%= link_to "Deploy All Projects", deploy_all_admin_deploy_group_path(@deploy_group), class: "btn btn-default", data: { method: "post" } %>
-      <% end %>
+        <% num_elligable_merge_stages = @deploy_group.stages.where.not(template_stage_id: nil).count %>
```

Comment:
`.where.not(template_stage_id: nil)` was reused twice, maybe turn it into a scope like `not_cloned` ?